README.md

```diff
@@ -8,19 +8,29 @@ doesn't handle is posting your files back to an issue for review.
 
 ## Installation
 
-Install dependencies with Composer:
+Dorgflow is designed to be installed globally.
+
+1. Either clone the repository or download a release and place it anywhere you
+like.
+
+2. Install dependencies with Composer:
 
```

Comment:
Here I would add something like: "Change to the directory where you clone or download the release and then run:"

Reply:
Done!